aw_watcher_afk/windows.py

```diff
@@ -0,0 +1,41 @@
+import time
+
+import ctypes
+from ctypes import Structure, POINTER, WINFUNCTYPE, windll
+from ctypes.wintypes import BOOL, UINT, DWORD
+
+
+class LastInputInfo(Structure):
+    _fields_ = [
+        ("cbSize", UINT),
+        ("dwTime", DWORD)
+    ]
+
+
+def _getLastInputTick():
+    prototype = WINFUNCTYPE(BOOL, POINTER(LastInputInfo))
+    paramflags = (1, "lastinputinfo"),
+    c_GetLastInputInfo = prototype(("GetLastInputInfo", ctypes.windll.user32), paramflags)
+
+    l = LastInputInfo()
+    l.cbSize = ctypes.sizeof(LastInputInfo)
+    assert 0 != c_GetLastInputInfo(l)
+    return l.dwTime
+
+
+def _getTickCount() -> int:
```

Comment:
We should probably use [GetTickCount64](https://msdn.microsoft.com/en-us/library/windows/desktop/ms724411(v=vs.85).aspx) instead.